brave/brave-core
# feat(wallet): ENS Off Chain Lookup in New Send Tab

## Description 
Implemented `ENS Off Chain Lookup` for the new `Send` tab


Resolves <https://github.com/brave/brave-browser/issues/27109>

## Submitter Checklist:


## Reviewer Checklist:


## After-merge Checklist:

  changes has landed on

## Test Plan:

### Testing Ethereum
1. Go to brave://settings/web3 and make sure `Allow ENS offchain lookup` is set to `Ask`
2. Go to brave://wallet/send, choose a token on the `Ethereum` network, enter an amount to send and paste in `offchainexample.eth`
3. You should see the `ENS Offchain` message
4. Make sure the `Learn more` link routes to https://github.com/brave/brave-browser/wiki/ENS-offchain-lookup
5. Click on the `Use ENS domain` button
6. The `Message` should disappear and resolve the address.

https://user-images.githubusercontent.com/40611140/207937782-89e24c56-e7f7-4580-83a0-351e27e73e7a.mov
